.circleci/config.yml

```diff
@@ -125,7 +125,7 @@ jobs:
       - checkout
       - run:
           name: backend lint
-          command: './scripts/backend-lint.sh'
+          command: 'make -j2 lint-go'
```

Comment:
`-j2` Utilising those two cpus we use on the circleci, in theory it should be better by the number of 2

Reply:
Had to revert that one :///, these tasks take way too much memory executed in parallel for our 4gb machines